zerver/webhooks/deskdotcom/doc.md

```diff
@@ -1,17 +1,17 @@
-1.  {!create-stream.md!}
+1. {!create-stream.md!}
```

Comment:
@sbansal1999 can you explain what you're achieving with these spacing edits? It might be nice to pull this out to a separate prep PR and so @laurynmm can review it without needing to look at the rest of the changes here.

Reply:
Opened #26484 .